php-checks/src/main/resources/org/sonar/l10n/php/rules/php/S2092.html

```diff
@@ -0,0 +1,22 @@
+<p>The "secure" attribute prevents cookies from being sent over plaintext connections such as HTTP, where they would be easily eavesdropped upon.
+Instead, cookies with the secure attribute are only sent over encrypted HTTPS connections.</p>
+<h2>Noncompliant Code Example</h2>
+<pre>
+; php.ini
+session.cookie_secure = 0 ; Noncompliant
+
+// in PHP code
+session_set_cookie_params($lifetime, $path, $domain, false); // Noncompliant, the last parameter means that the session cookie should not be secure
```

Comment:
session_set_cookie_params is not handled in the implementation.

Reply:
Indeed. Added